content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Kubernetes Services for Egress Traffic
+description: Shows how to configure Istio  Kubernetes External Services.
+keywords: [traffic-management,egress]
+weight: 60
+---
+
+Kubernetes [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+services and Kubernetes services with
+[Endpoints](https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors)
+let you create a local DNS _alias_ to an external service.
+This DNS alias has the same form as the DNS entries for local services, namely
+`<service name>.<namespace name>.svc.cluster.local`. DNS aliases provide _location transparency_ for your workloads:
+the workloads can call local and external services in the same way. If at some point in time you decide to deploy the
+external service inside your cluster, you can just update its Kubernetes service to reference the local version. The workloads will continue to operate without any change.
```

Comment:
Not clear on how to just update its kubernetes service?  do you mean instead of using endpoints, users can switch with service with selector now?

Reply:
@linsun Right, the users can switch to the service with selector, or to change the target of the ExternalName service to point to a service inside the cluster.